packages/iov-keycontrol/src/wallets/slip10wallet.ts

```diff
@@ -283,7 +283,10 @@ export class Slip10Wallet implements Wallet {
         signature = await Ed25519.createSignature(message, await Ed25519.makeKeypair(privkey));
         break;
       case Slip10Curve.Secp256k1:
-        signature = await Secp256k1.createSignature(message, privkey);
+        signature =
+          prehashType === PrehashType.Keccak256
+            ? await Secp256k1.createSignatureEth(message, privkey)
+            : await Secp256k1.createSignature(message, privkey);
```

Comment:
Hmm, this means we hash twice for `prehashType === Sha256` which is broken in the current code